### ISSUE TITLE
JBossNetworkAPI Integration Not Compatible with Ansible 2.9

### ISSUE THREAD

**Issue by sabre1041** [ansible-middleware/common#17]
##### SUMMARY
The functions of the JBossNetworkAPI module is only compatible with newer versions of Ansible and specifically does not support Ansible 2.9  

##### ISSUE TYPE
 - Bug Report

##### ANSIBLE VERSION
2.9

##### COLLECTION VERSION
1.1.3

##### STEPS TO REPRODUCE
Utilize JBossNetworkAPI modules when running Ansible 2.9

##### EXPECTED RESULTS
JBossNetworkAPI modules function as designed

##### ACTUAL RESULTS
```
TASK [redhat.eap.eap_utils : Download Red Hat JBoss Enterprise Application Platform] ***********************************************************************************************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: TypeError: load_file_common_arguments() got an unexpected keyword argument 'path'
fatal: [app1.q82ps.internal -> localhost]: FAILED! => {"changed": false, "module_stderr": "Traceback (most recent call last):\n  File \"/home/devops/.ansible/tmp/ansible-tmp-1695831399.1469228-130821-104932925011801/AnsiballZ_product_download.py\", line 102, in <module>\n    _ansiballz_main()\n  File \"/home/devops/.ansible/tmp/ansible-tmp-1695831399.1469228-130821-104932925011801/AnsiballZ_product_download.py\", line 94, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/home/devops/.ansible/tmp/ansible-tmp-1695831399.1469228-130821-104932925011801/AnsiballZ_product_download.py\", line 40, in invoke_module\n    runpy.run_module(mod_name='ansible_collections.redhat.runtimes_common.plugins.modules.product_download', init_globals=None, run_name='__main__', alter_sys=True)\n  File \"/usr/lib64/python3.6/runpy.py\", line 205, in run_module\n    return _run_module_code(code, init_globals, run_name, mod_spec)\n  File \"/usr/lib64/python3.6/runpy.py\", line 96, in _run_module_code\n    mod_name, mod_spec, pkg_name, script_name)\n  File \"/usr/lib64/python3.6/runpy.py\", line 85, in _run_code\n    exec(code, run_globals)\n  File \"/tmp/ansible_redhat.runtimes_common.product_download_payload_xollcu22/ansible_redhat.runtimes_common.product_download_payload.zip/ansible_collections/redhat/runtimes_common/plugins/modules/product_download.py\", line 300, in <module>\n  File \"/tmp/ansible_redhat.runtimes_common.product_download_payload_xollcu22/ansible_redhat.runtimes_common.product_download_payload.zip/ansible_collections/redhat/runtimes_common/plugins/modules/product_download.py\", line 253, in main\nTypeError: load_file_common_arguments() got an unexpected keyword argument 'path'\n", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}


```

**Comment by guidograzioli** [comment]
Also fixed by #19 ([86824176f5f720](https://github.com/ansible-middleware/common/pull/19/commits/86824176f5f72078b2e13465ca1637dda3fe34c2))

**Comment by sabre1041** [comment]
Fixed indeed by #19
